### PR TITLE
Update destination-functions.md

### PR DESCRIPTION
**Pull request by Hopes-Chantel**
I have added information about allowlisting to the FAQ section of these docs.

### CODE REVIEW

File: src/connections/functions/destination-functions.md [hunk 1]
```diff
@@ -419,6 +419,16 @@ No, destination functions are currently available as cloud-mode destinations onl
 
 If you are a partner, looking to publish your destination and distribute your app through Segment catalog, visit the [Developer Center](https://segment.com/partners/developer-center/){:target="_blank"} and check out the Segment [partner docs](/docs/partners/).
 
+##### Which IP addresses should be allowlisted?
+
+In terms of data leaving Segment’s servers to go to various destinations (not including warehouses), Segment uses Amazon Web Services (AWS), utilizing many different machines in order to send requests.
```
Comment (forstisabella) [inline src/connections/functions/destination-functions.md:424]: Why would someone allowlist IPs? @Hopes-Chantel 

```suggestion
You should allowlist the following [Amazon Web Services (AWS) IP addresses](https://ip-ranges.amazonaws.com/ip-ranges.json){:target="_blank"}:
```

File: src/connections/functions/destination-functions.md [hunk 2]
```diff
@@ -419,6 +419,16 @@ No, destination functions are currently available as cloud-mode destinations onl
 
 If you are a partner, looking to publish your destination and distribute your app through Segment catalog, visit the [Developer Center](https://segment.com/partners/developer-center/){:target="_blank"} and check out the Segment [partner docs](/docs/partners/).
 
+##### Which IP addresses should be allowlisted?
+
+In terms of data leaving Segment’s servers to go to various destinations (not including warehouses), Segment uses Amazon Web Services (AWS), utilizing many different machines in order to send requests. 
+
+These IP addresses can be found [here:](https://ip-ranges.amazonaws.com/ip-ranges.json). If you’d like to do so, you will need to allowlist all the IPs from your workspace’s location range.
```
Comment (forstisabella) [inline src/connections/functions/destination-functions.md:426]: Do you have to allowlist all IP addresses for the workspace location range?

**Comment by forstisabella** [comment]
@stayseesong - looks like this and #5238 are the same content in two different locations - should we consolidate these PRs into one `include`?

**Comment by stayseesong** [comment]
@forstisabella sure we can do that!

**Comment by forstisabella** [comment]
Hi @Hopes-Chantel! We created a reusable snippet to make sure that the info on the destination-functions page and the destination overview page was the same: if you wanted to edit this in the future, you can find the snippet at `/src/_includes/content/ip-allowlist.md`!

**Pull request closed without merging**